### DISCUSSION TITLE
Numerical tracker reducer

### DISCUSSION THREAD

**Discussion by rzarai** [iSoron/uhabits#1126] [Feature Requests] [answered]
Currently we have a numerical tracker added not so far ago, but it tracks in increasing order, as I can see. There is no way you can track the numerical amount of something in reducing manner.  
I mean for example you spend hours and hours watching youtube. And you want to bring this number from 4-6 hours a day, down to two hours a day. That would be neat feature indeed.  
  
So in short what I mean is to set goal aiming on reducing numerical (or temporal) amount of something to spend less time on something with each passing day.

**Comment by KristianTashkov** [comment, marked as answer]
This is a duplicate of https://github.com/iSoron/uhabits/discussions/949.
Implementation is in review here so it could be available in the application soon: https://github.com/iSoron/uhabits/pull/1101
